Samples/V1.0Samples/LocalMediaSamples/AudioVideoPlaybackBot/ADF/bicep/global/prefix.json

```diff
@@ -0,0 +1,422 @@
+{
+    "AFS1": {
+        "displayname": "France South",
+        "location": "francesouth",
+        "first": "France",
+        "second": "South",
+        "third": null,
+        "Name": "FS1",
+        "NameOverRide": null,
+        "PREFIX": "AFS1"
+    },
+    "AAS1": {
```

Comment:
can we make ADF config files the mininum set

Reply:
If you don't strongly feel about this, I wanted to keep these prefix list so that if the sample is used in any azure region it is supported. I will talk to the ADF team to understand how we can remove these and probably pull them on runtime as a follow up.